### PR TITLE
Added create key method to security_keyvault

### PR DESCRIPTION
**Pull request by daniel-larsen**

### CODE REVIEW

File: sdk/security_keyvault/src/models.rs
[the remark this answers is not in the document]
Comment (cataggar) [reply sdk/security_keyvault/src/models.rs:1]: I'm indifferent. You could reexport it.

**Review by cataggar — COMMENTED** [review]
@heaths, you are the subject matter expert. Can you guide this to completion?

**Comment by heaths** [comment]
> @heaths, you are the subject matter expert. Can you guide this to completion?

I'm waiting to see requested changes.

**Comment by cataggar** [comment]
Please reopen if this is picked back up.

**Pull request closed without merging**